hkm/templatetags/hkm_tags.py

```diff
@@ -35,14 +36,16 @@ def record_detail(record):
     format = record.get('rawData').get('format', [])
     measures = record.get('rawData').get('measurements', [])
 
-    for type in format:
-        translated.append(type.get('translated'))
+    for data in format:
+        if type(data) is dict:
```

Comment:
```suggestion
        if isinstance(data, dict):
```
Doesn't really matter in this case, but generally `isinstance` is preferred over `type` when doing type checks.

Reply:
Ah, I'll change this.

About the logging: It seems that at the moment all of the images are throwing this error, I'm just wondering if it's worth it to log the errors as it would result in hundreds of lines. My plan of attack was to implement a quick fix and deploy it into production, while I email Finna about this (no changes were made to API documentation at least). The reason I'm not removing this all together is that if the cause is on their end and it's behaviour is returned to normal, I don't need to make another change.